### ISSUE TITLE
Presentation Exchange: Submission Requirements Features

### ISSUE THREAD

**Issue by decentralgabe** [TBD54566975/did-sdk#56]
Feature from https://identity.foundation/presentation-exchange/#features

**Comment by andorsk** [comment]
@decentralgabe isn't this already finished?

**Comment by decentralgabe** [comment]
no. the model is there, but the processing is not https://github.com/TBD54566975/ssi-sdk/blob/main/credential/exchange/submission.go#L487

**Comment by andorsk** [comment]
ahhh. thanks for the update.
